src/cms/views/users/region_user_view.py

```diff
@@ -94,6 +94,9 @@ def post(self, request, *args, **kwargs):
             request.POST, instance=user_profile_instance
         )
 
+        if user_profile_instance and user_profile_instance.send_activation_link:
```

Comment:
```suggestion
        if user_instance and user_instance.active:
```
If you go with my suggestion of removing the `send_activation_link` field from the database, you can also use the `active` field here.